artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueManagerImpl.java

```diff
@@ -34,15 +37,13 @@ private void doIt() {
       Queue queue = server.locateQueue(queueName);
       //the queue may already have been deleted and this is a result of that
       if (queue == null) {
-         if (ActiveMQServerLogger.LOGGER.isDebugEnabled()) {
-            ActiveMQServerLogger.LOGGER.debug("no queue to delete \"" + queueName + "\".");
+         if (logger.isDebugEnabled()) {
```

Comment:
Why change of logger now means the loss of formatted/and id'd log

Reply:
this is logger.debug...

anything related to debug and tracing should be done through an individual logger... Logger.getLogger(className).

this was a mistake